FALSEHOODS.md

```diff
@@ -0,0 +1,79 @@
+# Falsehoods Programmers Believe About Phone Numbers
+
+... and tips on how to use libphonenumber.
+
+Given how ubiquitous phone numbers are and how long they've been around, it's surprising how many false assumptions programmers continue to make about them.
+
+1.  **Phone numbers that are valid today will always be valid. Phone numbers of a certain type today (e.g., mobile) will never be reassigned to another type.**
+
+    A phone number which connects today may be disconnected tomorrow. A number which is free to call today may cost money to call tomorrow. The phone company may decide to expand the range of available phone numbers by inserting a digit into an existing number.
+
+    **Tip:** Don’t store properties for a phone number such as validity or type. Check this information again from the library when you need it.
+
+1.  **A phone number uniquely identifies an individual**
+
+    It wasn't even that long ago that mobile phones didn't exist, and it was common for an entire household to share one fixed-line telephone number. In some parts of the world, this is still true, and relatives (or even friends) share a single phone number.
+
+1.  **An individual has only one phone number**
+
+    Obviously, this isn't necessarily true.
+
+1.  **Phone numbers cannot be re-used**
+
+    Old phone numbers are recycled and get reassigned to other people.
+
+1.  **Each country calling code corresponds to exactly one country**
+
+    The USA, Canada, and several Caribbean islands share the country calling code +1. Russia and Kazakhstan share +7. These are not the only examples!
+
+1.  **Each country has only one country calling code**
+
+    As of this present moment (in Mar. 2016), phones in the disputed territory and partially recognised state of Kosovo may be reached by dialing the country calling code for Serbia (+381), Slovenia (+386), or Monaco (+377), depending on where and when one obtained the number.
+
+    *Tip:* Use the phone widget to encourage users to enter their phone number in an international format such that we can understand it.
```

Comment:
Bold instead of italic? To be consistent with the rest (can be a separate patch)